### PR TITLE
HTML pieces move into classes wrapping the source HTML files. This al…

### PR DESCRIPTION
**Pull request by kozakHolota**
…lows us to edit/view them in a browser.

# Merge checklist
- [x] TravisCI tests passed
- [x] Documentation

### CODE REVIEW

**Comment by prashanth-sams** [comment]
Thank you for the PR @kozakHolota - I will take a look on this today. Can you please add me in twitter (@prashanthsams), to get you in our dev slack.

**Review by prashanth-sams — CHANGES_REQUESTED** [review]
@kozakHolota some UI changes required
https://user-images.githubusercontent.com/2948696/159121848-d13a1e48-a510-4bde-a5e5-d7bd86deae8f.mov

**Comment by kozakHolota** [comment]
@prashanth-sams
Fixed the HTML layout. Tested on my own. Layout looks good. Please check.

**Review by prashanth-sams — CHANGES_REQUESTED** [review]
@kozakHolota awesome work with lots of fixes; I still see few UI improvements needed. You can compare with the old reports if you have - else I can share.
Screen Shot 2022-03-20 at 7 13 06 PM

Screen Shot 2022-03-20 at 7 13 26 PM


https://user-images.githubusercontent.com/2948696/159169377-04f42708-533d-4abd-8b30-1db4d22e5e80.mov

**Comment by kozakHolota** [comment]
> @kozakHolota awesome work with lots of fixes; I still see few UI improvements needed. You can compare with the old reports if you have - else I can share. Screen Shot 2022-03-20 at 7 13 06 PM
> 
> Screen Shot 2022-03-20 at 7 13 26 PM
> 
>  Screen.Recording.2022-03-20.at.7.12.31.PM.mov

The problem here was the global vars usage which makes any refactoring quite difficult business. For now I've solved this moving all global vars into the separate module `pytest_html_reporter.const_vars`. For the future I think we need to move them into some _class constants_ or _Enum members_ to make possible any **extension** of this product. So, Please review and provide your feedback.

**Comment by kozakHolota** [comment]
image
Style is fixed. Now everything looks very well. Please review.

**Comment by prashanth-sams** [comment]
@kozakHolota Nice; btw, can you apply the same style fixes in 'test suite' as well? 
image

also, anything we can do in archive layout? cos it is an important feature in our report

**Comment by kozakHolota** [comment]
@prashanth-sams  - fixed archive rows. Pease review.

**Review by kozakHolota — COMMENTED** [review]
Archive rows are fixed. Please review.

**Comment by prashanth-sams** [comment]
@kozakHolota Impressive! one last change needed :D 

It's with the title
```
pytest tests/ --html-report=./report --title='XYZ'
```

image

File: html/screenshot_details.html [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<div class="img-hover col-md-6 col-xl-3 p-3">
```
Comment (prashanth-sams) [inline html/screenshot_details.html:1]: indentation

File: html/floating_error.html [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<a data-toggle="modal" href="#myModal-__runt__" class="">(...)</a>
```
Comment (prashanth-sams) [inline html/floating_error.html:1]: indentation

File: html/archive_row.html [hunk 3]
```diff
@@ -0,0 +1,5 @@
+<a class ="list-group-item list-group-item-action" href="#list-item-__acount__" style="font-size: 1.1rem; color: dimgray; margin-bottom: -7%;">
+                        <i class="fa fa-__astate__" aria-hidden="true" style="color: __astate_color__"></i>
```
Comment (prashanth-sams) [inline html/archive_row.html:2]: indentation

File: html/archive_body.html [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+                                                <p class="stats-text">PASSED</p>
+                                            </div>
+                                        </div>
+                                        <div class="col-xs-12 col-md-3" style="max-width: 14.2%;">
+                                            <div class="counter">
+                                                <h2 class="timer count-title count-number">__fail__
+                                                </h2>
+                                                <p class="stats-text">FAILED</p>
+                                            </div>
+                                        </div>
```
Comment (prashanth-sams) [inline html/archive_body.html:40]: indentation

File: setup.py [hunk 5]
```diff
@@ -13,7 +13,7 @@ def read(fname):
 
 setup(
     name="pytest-html-reporter",
-    version="0.2.9",
+    version="0.3",
```
Comment (prashanth-sams) [inline setup.py:16]: Let's make it `0.3.0`

**Review by prashanth-sams — CHANGES_REQUESTED** [review]
custom title fix needed + minor indentations

File: html/archive_body.html (hunk 4 above)
Comment (kozakHolota) [reply html/archive_body.html:40]: Could you please let me know what do you mean under _identation_ ?

Comment (prashanth-sams) [reply html/archive_body.html:40]: @kozakHolota shifting the html code to left with better alignment

Comment (prashanth-sams) [reply html/archive_body.html:40]: Custom title is something important to fix and apart from that we are good

**Comment by kozakHolota** [comment]
> custom title fix needed + minor indentations
Everything is fixed.

image

**Review by prashanth-sams — APPROVED** [review]
All Good and awesome work!

**Pull request merged**